packages/taquito-core/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@taquito/core",
+  "version": "16.1.2",
+  "description": "high level error classes and interfaces",
```

Comment:
Maybe: `Classes, interfaces, and types shared between many other Taquito packages, including error types`